src/main/java/com/codeborne/selenide/impl/ScreenShotLaboratory.java

```diff
@@ -332,12 +341,37 @@ public void startContext(String context) {
     }
   }
 
+  public List<File> getThreadScreenshots() {
+    return Collections.unmodifiableList(threadScreenshots.get());
+  }
+
+  public List<File> getContextScreenshots() {
+    List<File> screenshots = currentContextScreenshots.get();
+    return Collections.unmodifiableList(screenshots == null
+      ? Collections.emptyList()
+      : screenshots);
+  }
```

Comment:
I would change the order: there is no need to call `unmodifiableList` for `emptyList`:

```
return screenshots == null
      ? Collections.emptyList()
      : Collections.unmodifiableList(screenshots);
```